lib/elasticity/index_mapper.rb

```diff
@@ -71,10 +72,16 @@ def remap!(retry_delete_on_recoverable_errors: true, retry_delay: 30, max_delay:
     end
 
     # Flushes the index, forcing any writes
+    # note that v7 no longer forces any writes on flush
     def flush_index
```

Comment:
would emitting a deprecation log warning be useful here, or is `flush_index` still have value if it doesn't force the writes?

Reply:
`flush` is still a valid `thing`
https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-flush.html#flush-api-desc

 it just want have the side-effect of refreshing the index which made new documents visible to searches (the side-effect that is has pre v7)